precompile/assetsbridge/byte_code.go

```diff
@@ -14,4 +14,4 @@ package assetsbridge
1525f906012611edf60921b019063ff39fd04906105b59089908990899089908990600401610ae1565b6020604051808303815f875af11580156105d1573d5f803e3d5ffd5b505050506040513d601f19601f820116820180604052508101906105f59190610920565b9695505050505050565b6001600160a01b0381168114610613575f80fd5b50565b5f60208284031215610626575f80fd5b8135610296816105ff565b5f8060408385031215610642575f80fd5b823561064d816105ff565b9150602083013561065d816105ff565b809150509250929050565b60408101610216828480516001600160a01b03908116835260209182015116910152565b602080825282518282018190525f919060409081850190868401855b828110156106e1576106d184835180516001600160a01b03908116835260209182015116910152565b92840192908501906001016106a8565b5091979650505050505050565b5f80604083850312156106ff575f80fd5b823561070a816105ff565b946020939093013593505050565b634e487b7160e01b5f52604160045260245ffd5b6040516080810167ffffffffffffffff8111828210171561074f5761074f610718565b60405290565b604051601f8201601f1916810167ffffffffffffffff8111828210171561077e5761077e610718565b604052919050565b5f67ffffffffffffffff82111561079f5761079f610718565b5060051b60200190565b5f60208083850312156107ba575f80fd5b823567ffffffffffffffff8111156107d0575f80fd5b8301601f810185136107e0575f80fd5b80356107f36107ee82610786565b610755565b81815260079190911b82018301908381019087831115610811575f80fd5b928401925b82841015610879576080848903121561082d575f80fd5b61083561072c565b8435815285850135610846816105ff565b8187015260408581013590820152606080860135610863816105ff565b9082015282526080939093019290840190610816565b979650505050505050565b5f805f805f60808688031215610898575f80fd5b85356108a3816105ff565b945060208601359350604086013560ff811681146108bf575f80fd5b9250606086013567ffffffffffffffff808211156108db575f80fd5b818801915088601f8301126108ee575f80fd5b8135818111156108fc575f80fd5b89602082850101111561090d575f80fd5b9699959850939650602001949392505050565b5f60208284031215610930575f80fd5b81518015158114610296575f80fd5b5f6020828403121561094f575f80fd5b8151610296816105ff565b5f6020828403121561096a575f80fd5b5051919050565b5f60408284031215610981575f80fd5b6040516040810181811067ffffffffffffffff821117156109a4576109a4610718565b806040525080915082516109b7816105ff565b815260208301516109c7816105ff565b6020919091015292915050565b5f604082840312156109e4575f80fd5b6102968383610971565b5f60208083850312156109ff575f80fd5b825167ffffffffffffffff811115610a15575f80fd5b8301601f81018513610a25575f80fd5b8051610a336107ee82610786565b8082825260208201915060208360061b850101925087831115610a54575f80fd5b6020840193505b8284101561087957610a6d8885610971565b82528482019150604084019350610a5b565b602080825282518282018190525f919060409081850190868401855b828110156106e157815180518552868101516001600160a01b03908116888701528682015187870152606091820151169085015260809093019290850190600101610a9b565b6001600160a01b03861681526020810185905260ff841660408201526080606082018190528101829052818360a08301375f81830160a090810191909152601f909201601f1916010194935050505056fea264697066735822122017b065e2ebefe19fef656645cf1a90c32147bff49b372eddafd020f9ffd5008c64736f6c63430008180033"
```

Comment:
Copied from `deployedBytecode` (without the `0x` prefix) from `mezod/precompile/hardhat/artifacts/contracts/AssetsBridgeCaller.sol/AssetsBridgeCaller.json` after running `npx hardhat compile`.

Reply:
looks good to me.